.github/workflows/compilers.yml

```diff
@@ -63,154 +62,155 @@ jobs:
-          - { key: default_cc, name: clang-4.0, value: clang-4.0, container: clang-4.0 }
-          - { key: default_cc, name: clang-3.9, value: clang-3.9, container: clang-3.9 }
-          - key: default_cc
-            name: 'clang-14 LTO'
-            value: 'clang-14 -flto=auto'
+          - { name: clang-15,  env: { default_cc: clang-15 } }
+          - { name: clang-14,  env: { default_cc: clang-14 } }
+          - { name: clang-13,  env: { default_cc: clang-13 } }
+          - { name: clang-12,  env: { default_cc: clang-12 } }
+          - { name: clang-11,  env: { default_cc: clang-11 } }
+          - { name: clang-10,  env: { default_cc: clang-10 } }
+          - { name: clang-9,   env: { default_cc: clang-9 } }
+          - { name: clang-8,   env: { default_cc: clang-8 } }
+          - { name: clang-7,   env: { default_cc: clang-7 } }
+          - { name: clang-6.0, env: { default_cc: clang-6.0 } }
+          - { name: clang-5.0, env: { default_cc: clang-5.0 } }
+          - { name: clang-4.0, env: { default_cc: clang-4.0 } }
+          - { name: clang-3.9, env: { default_cc: clang-3.9 } }
+          - name: 'clang-14 LTO'
             container: clang-14
-            configure_append: '--disable-shared optflags=-O2'
+            env:
+              default_cc: 'clang-14 -flto=auto'
+              optflags: '-O2'
+            shared: disable
             # check: true
 
-#         - { key: crosshost, name: aarch64-linux-gnu,     value: aarch64-linux-gnu, container: crossbuild-essential-arm64 }
-#         - { key: crosshost, name: arm-linux-gnueabi,     value: arm-linux-gnueabi }
-#         - { key: crosshost, name: arm-linux-gnueabihf,   value: arm-linux-gnueabihf }
-#         - { key: crosshost, name: i686-w64-mingw32,      value: i686-w64-mingw32 }
-#         - { key: crosshost, name: powerpc-linux-gnu,     value: powerpc-linux-gnu }
-#         - { key: crosshost, name: powerpc64le-linux-gnu, value: powerpc64le-linux-gnu, container: crossbuild-essential-ppc64el }
-#         - { key: crosshost, name: s390x-linux-gnu,       value: s390x-linux-gnu, container: crossbuild-essential-s390x }
-#         - { key: crosshost, name: x86_64-w64-mingw32,    value: x86_64-w64-mingw32, container: mingw-w64 }
+#         - { name: aarch64-linux-gnu,     crosshost: aarch64-linux-gnu, container: crossbuild-essential-arm64 }
+#         - { name: arm-linux-gnueabi,     crosshost: arm-linux-gnueabi }
+#         - { name: arm-linux-gnueabihf,   crosshost: arm-linux-gnueabihf }
+#         - { name: i686-w64-mingw32,      crosshost: i686-w64-mingw32 }
+#         - { name: powerpc-linux-gnu,     crosshost: powerpc-linux-gnu }
+#         - { name: powerpc64le-linux-gnu, crosshost: powerpc64le-linux-gnu, container: crossbuild-essential-ppc64el }
+#         - { name: s390x-linux-gnu,       crosshost: s390x-linux-gnu, container: crossbuild-essential-s390x }
+#         - { name: x86_64-w64-mingw32,    crosshost: x86_64-w64-mingw32, container: mingw-w64 }
```

Comment:
Maybe we can just delete this part...  It seems `crosshost` is just making things complicated.

Reply:
Remove cross-compilation checks all?

Reply:
All from this file.  They are commented out due to https://github.com/ruby/ruby/pull/5693 .  I don't think we can fix the issue soon.

Reply:
When I tried recently (https://github.com/nobu/ruby/actions/runs/2534101312), thees seem working?

Reply:
Seems not working? https://github.com/nobu/ruby/runs/6980800149?check_suite_focus=true#step:12:1011